src/config/entities/__tests__/configuration.ts

```diff
@@ -20,6 +21,9 @@ export default (): ReturnType<typeof configuration> => ({
     default: faker.number.int(),
   },
   httpClient: { requestTimeout: faker.number.int() },
+  log: {
+    level: sample(['error', 'warn', 'info', 'debug']) ?? 'debug',
```

Comment:
What use case did you consider to sample from different levels for logging in the tests?

Reply:
I took inspiration from `npm` logging levels described [here](https://github.com/winstonjs/winston#logging-levels) but I didn't take all of them for the sake of brevity. Maybe it's better to sample from all the _allowed_ values (related to https://github.com/safe-global/safe-client-gateway-nest/pull/516#discussion_r1245332342)

Reply:
> I took inspiration from `npm` logging levels described [here](https://github.com/winstonjs/winston#logging-levels) but I didn't take all of them for the sake of brevity. Maybe it's better to sample from all the _allowed_ values (related to [#516 (comment)](https://github.com/safe-global/safe-client-gateway-nest/pull/516#discussion_r1245332342))

My question was not about what different logging levels exist but why do we need to sample between logging levels. What use case did you consider?

Reply:
My initial thought was to provide realistic random values to test classes as we do with the rest of the test configuration properties, but maybe this doesn't make a lot of sense and we could have a fixed value here. In any case, I think this value wouldn't be used in any place since the mocked `ILoggingService` we use in tests [redirects to the console](https://github.com/safe-global/safe-client-gateway-nest/blob/c4e872915755574a83a18efeee91172713dc6167/src/logging/__tests__/test.logging.module.ts#L4). Correct me if I'm wrong 🙂

Reply:
I think we should set a default value (`debug` for example) else it can be confusing when you run your tests multiple times.

Logs that are generated by tests can also be useful for debugging. If the logging level is random, then we generate different logs on different runs for the same codebase.

Reply:
Agree, changed in [317288e](https://github.com/safe-global/safe-client-gateway-nest/pull/516/commits/317288eac9dc6f63ee3f853a6c7dab6169e50db9)